pkg/syncthing/k8s/crud.go

```diff
@@ -20,8 +20,9 @@ var devTerminationGracePeriodSeconds int64
 func Deploy(dev *model.Dev, d *appsv1.Deployment, c *apiv1.Container, client *kubernetes.Clientset) error {
 	ss := translate(dev, d, c)
 
-	if exists(ss, client) {
+	if old := exists(ss, client); old != nil {
 		ss.Spec.Template.Spec.NodeName = pods.GetSyncNode(dev, client)
+		ss.Spec.PodManagementPolicy = old.Spec.PodManagementPolicy
```

Comment:
This is to avoid forcing `okteto down -v` if you already have a dev environment running, since changing `PodManagementPolicy` is an incompatible statefulset change

Reply:
When would `okteto down -v` be required?

Is there a way to catch the error message and tell the user when it's needed?

Reply:
It is not required because we only do this for new `okteto up`s